src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_VALUE = 100;
```

Comment:
```suggestion
    private static final int MAX_BALL_NUMBER = 100;
```
Try to use more informative names for your variables.